scripts/feature_tests/upgrade/upgrade_vars.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -x
+
+export CAPM3RELEASE="v0.3.2"
+export CAPIRELEASE="v0.3.4"
```

Comment:
this does not look like it is used

Reply:
I mean there is nothing sourcing this file as far as I can see

Reply:
true, the usage of non-default CAPM3RELEASE and CAPIRELEASE values sre commented out temporarily and this file is kept on purpose.